README.md

```diff
@@ -469,5 +469,38 @@ WhereOrder: LIKE filter in line 8 precedes a more selective filter.
 ```
 
 
+
+## Anti Pattern 10: Using an analytic functions over self-joins
+Example:
```

Comment:
Can we please use a simpler example here

```
select 
  t1.*, 
  t2.value as prev_value
from 
  table t1
left join 
  table t2
  on 
    t1.id = t2.id
    and t1.date - 1 = t2.date
```

Reply:
Gave simple examples